overrides/cf.d.ts

```diff
@@ -152,6 +152,51 @@ interface RequestInitCfPropertiesImageDraw extends BasicImageTransformations {
    * overlays we recommend using PNG or WebP images.
    */
   url: string;
+  /**
+   * Maximum width of the overlay image, in pixels. It must be an integer.
+   */
+  width?: number;
```

Comment:
All these types in `RequestInitCfPropertiesImageDraw` should be inherited from `BasicImageTransformations` (see L148).

Reply:
You are right, as it extends `BasicImageTransformations`. Let me fix that.

Reply:
Should be fixed.